python-spec/src/somacore/query/query.py

```diff
@@ -472,6 +535,14 @@ class _AxisQueryResult:
     """Experiment.ms[...].X[...] query slice, as an SciPy sparse.csr_matrix """
     X_layers: Dict[str, sparse.csr_matrix] = attrs.field(factory=dict)
     """Any additional X layers requested, as SciPy sparse.csr_matrix(s)"""
+    obsm: Dict[str, sparse.csr_matrix] = attrs.field(factory=dict)
```

Comment:
Originally, `_AxisQueryResult` structure was pure Arrow. If it is now a mismash of stuff for use in (only) `to_anndata`, we should rename and clean it up so that it is so-documented.